package.json

```diff
@@ -28,5 +28,8 @@
         "prettier": "^2.3.2",
         "ts-node": "^10.5.0",
         "typescript": "^4.2.2"
+    },
+    "dependencies": {
+        "@changesets/cli": "^2.27.1"
```

Comment:
Please move this to dev dependencies (this is kinda wrong in Comet)

Reply:
done in [bc54768d90c671562114c5067325b4e74b04973e](https://github.com/vivid-planet/comet-brevo-module/pull/11/commits/bc54768d90c671562114c5067325b4e74b04973e)